entity-server-api/pom.xml

```diff
@@ -40,5 +40,10 @@
       <artifactId>entity-common-api</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
this new dependency worries me a bit.  I wonder if a new exception type should be defined on the server-side that gets translated to EntityUserException on the client side.